futurenhs.automation/features/userAccess.feature

```diff
@@ -1,111 +1,112 @@
 Feature: User Access
     User journeys around users accessing the portal, including login page, registration, and unauthenticated page restrictions.
 
-Background:
-    Given I have navigated to '/'
+    Background:
+        Given I have navigated to '/'
```

Comment:
Is there a particular reason as to why you are doing a lot of formatting changes?

Reply:
Mainly consistency and so it's easier to read. It's "cucumber formatter" in vscode.